### DISCUSSION TITLE
Flickering during search

### DISCUSSION THREAD

**Discussion by ATM-Jahid** [LazyVim/LazyVim#3355] [General]
I have been noticing flickers in neovim (with lazyvim) when I search for words. This might be due to redrawing of the screen. I don't notice this with `nvim --clean`.

Is this a known issue? Is there any way to get rid of the flickers by disabling any specific plugin?

**Comment by folke** [comment]
Known issue for Noice.
The issue has actually been fixed and merged today in Neovim, so I'll make a new Noice release that removes those needed redraws.

- [ ] https://github.com/neovim/neovim/pull/27950
